packages/synapse-react-client/src/components/dataaccess/UserAccessRequestHistory/UserAccessRequestHistoryPlace.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  BreadcrumbItem,
+  Breadcrumbs,
+} from '@/components/Breadcrumbs/Breadcrumbs'
+import { Box } from '@mui/material'
+import { useMemo } from 'react'
+import {
+  createBrowserRouter,
+  createMemoryRouter,
+  Navigate,
+  Outlet,
+  RouteObject,
+  RouterProvider,
+  useLocation,
+  useParams,
+} from 'react-router'
+import { RouterProvider as DOMRouterProvider } from 'react-router/dom'
+import SubmissionPage from '../SubmissionPage/SubmissionPage'
+import {
+  SUBMISSION_ID_PARAM,
+  USER_ACCESS_HISTORY_SUBMISSION_SUBPATH,
+} from './RouteConstants'
+import { UserAccessRequestHistoryPage } from './UserAccessRequestHistoryPage'
+
+type ReviewerDashboardProps = {
+  /** Used to determine the base path for the component. Default is RequestHistory:default */
+  routerBaseName?: string
+  /** If true use a MemoryRouter, which prevents the browser URL from updating. For demo/test purposes only. */
+  useMemoryRouter?: boolean
+}
+
+function UserSubmissionPageRouteRenderer() {
+  const { submissionId } = useParams<{ [SUBMISSION_ID_PARAM]: string }>()
+  if (!submissionId) {
+    return <Navigate to="/" />
+  }
+  return <SubmissionPage submissionId={submissionId} isReviewer={false} />
+}
+
+/**
+ * Layout for UserAccessRequestHistory pages. Includes breadcrumbs and a header with title.
+ * @constructor
+ */
+function UserAccessRequestHistoryLayout() {
+  const { submissionId } = useParams<{ [SUBMISSION_ID_PARAM]: string }>()
+  const location = useLocation()
+
+  const breadcrumbItems: BreadcrumbItem[] = [
+    {
+      text: 'Access Request History',
+      to: '/',
```

Comment:
I feel like I'm missing something.  Should this root breadcrumb item point to routerBaseName?

Reply:
Or is this already relative to that routerBaseName?

Reply:
Correct.